mappings/net/minecraft/client/sound/MusicType.mapping

```diff
@@ -1 +1,14 @@
-CLASS net/minecraft/class_1143 net/minecraft/client/sound/MusicType
+CLASS net/minecraft/class_5195 net/minecraft/client/sound/MusicType
+	FIELD field_24057 soundEvent Lnet/minecraft/class_3414;
+	FIELD field_24058 minDuration I
+	FIELD field_24059 maxDuration I
+	FIELD field_24060 replaceSong Z
+	METHOD <init> (Lnet/minecraft/class_3414;IIZ)V
+		ARG 1 soundEvent
+		ARG 2 minDuration
+		ARG 3 maxDuration
+		ARG 4 replaceSong
+	METHOD method_27279 getSoundEvent ()Lnet/minecraft/class_3414;
+	METHOD method_27280 getMinDuration ()I
+	METHOD method_27281 getMaxDuration ()I
+	METHOD method_27282 shouldReplaceSong ()Z
```

Comment:
Is "song" something different from music? If not, I think "shouldReplaceMusic" would be clearer. Also, if this returns false, do two songs get played at once? I don't really see how that's useful.

Reply:
Oh yeah, that was meant to be `music`. If it returns false, the song doesn't play.